### ISSUE TITLE
gtouch (coreutils 8.28) fails after upgrade to Xcode 9

### ISSUE THREAD

**Issue by sramsay** [Homebrew/homebrew-core#18380]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

# Please always follow these steps:
- [X] Confirmed this is a problem with `brew install`ing one, specific Homebrew/homebrew-core formula (not cask or tap) and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Homebrew/brew/issues/new. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/caskroom/homebrew-cask/new. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [X ] Ran `brew update` and retried your prior step?
- [X ] Ran `brew doctor`, fixed all issues and retried your prior step?
- [X] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?

https://gist.github.com/sramsay/5a8612076774b4fe1659ceab5717f03e

- [X] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)

Trying to use gtouch (from coreutils 8.28), in any context -- including using brew itself.

- What happened (include command output)

Ever since updating to Xcode 9 the other day, I've had the following failure with gtouch (coreutils 8.28):

dyld: lazy symbol binding failed: Symbol not found: _futimens
  Referenced from: /usr/local/opt/coreutils/libexec/gnubin/touch
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _futimens
  Referenced from: /usr/local/opt/coreutils/libexec/gnubin/touch
  Expected in: /usr/lib/libSystem.B.dylib

Abort trap: 6

- What you expected to happen

Update  the access and modification times of each FILE to the current time.

Since touch is used so widely in homebrew itself (and in most compile processes), this has effectively broken my system.

- Step-by-step reproduction instructions (by running `brew install` commands)

This happens with *any* use of brew install (as well as any use gtouch), but the problem is clearly with coreutils.  Sorry, I wasn't quite sure which issue system to file this under.

# Formula additions or changes
To get formulae added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
We will close issues requesting formulae changes.

**Comment by ilovezfs** [comment]
```
HOMEBREW_VERSION: 1.3.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: 751334a257d81851e68da7ab390982d4e9fdf909
Last commit: 2 weeks ago
```

This should be fixed in 1.3.3. Please `brew update`.

**Comment by sramsay** [comment]
Excellent.  Thanks.
